jetty-core/jetty-core-server/src/main/java/org/eclipse/jetty/core/server/HttpChannel.java

```diff
@@ -404,9 +405,8 @@ public InvocationType getInvocationType()
             if (metaData == null)
                 new Throwable().printStackTrace();
             Objects.requireNonNull(metaData);
-            _requests++;
+            _requestId = ++_requests;
```

Comment:
It is thread safe because a HttpChannel is only used for one stream and thus cannot be called concurrently.  Specifically it is protected with:
```java

            if (_request != null)
                throw new IllegalStateException();
            _request = new ChannelRequest(request);
```

I've made it a long